.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build project with Maven
+on:
+  pull_request:
+  push:
+    branches-ignore: [master]
+  schedule:
+  - cron: '2 2 * * 1-5' # run nightly master builds on weekdays
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@e9512a931da86017e9d4d564a24616d8444d2193 # pin@v2
```

Comment:
same as in https://github.com/camunda/camunda-acm-plugin/pull/41: 
shall we consider `actions/*` trustworthy?